src/App.js

```diff
@@ -1,12 +1,74 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonTile from "./components/PokemonTile";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [error, toggleError] = useState(false);
+    const [loading, toggleLoading] = useState(false);
+    const [pokemonsData, setPokemonsData] = useState([]);
```

Comment:
Als `pokemonsData` nooit een array zal zijn, dan zou ik dat ook niet als default waarde pakken. Zo te zien zal het een object gaan zijn. Dan zou ik kiezen voor `undefined` als default waarde. Dan heb je een duidelijk onderscheid tussen of het al is ingeladen of niet.
```suggestion
    const [pokemonsData, setPokemonsData] = useState();
```

Reply:
Hee Zwen, dank voor je feedback!
Als ik hier de useState() op undefined zet, zoals je voorstelt. Krijg ik undefined errors, in dit geval mbt de pokemonsData.previous (Cannot read properties of undefined (reading 'previous')). Ik heb er nu ({}) van gemaakt, dan krijg ik geen errors.